build.gradle

```diff
@@ -1038,9 +1060,7 @@ project(':iceberg-spark3-runtime') {
     relocate 'com.google', 'org.apache.iceberg.shaded.com.google'
     relocate 'com.fasterxml', 'org.apache.iceberg.shaded.com.fasterxml'
     relocate 'com.github.benmanes', 'org.apache.iceberg.shaded.com.github.benmanes'
-    relocate 'org.antlr.v4.runtime', 'org.apache.iceberg.shaded.org.antlr.v4.runtime'
     relocate 'org.checkerframework', 'org.apache.iceberg.shaded.org.checkerframework'
-    relocate 'org.apache.avro', 'org.apache.iceberg.shaded.org.apache.avro'
```

Comment:
why remove this?

Reply:
oops that's a typo, I had alphabetized this list and when I undid it I must have missed this

Reply:
Looks like this hasn't been reverted yet.

Reply:
This original comment was for a missing relocate "avro" lib. The Antlr library I moved to the bottom since I feel like it's another separate category